README.md

```diff
@@ -111,6 +111,34 @@ await($pool);
 If an exception is thrown from within a child process, and not caught using the `->catch()` callback,
 it will be thrown as `Spatie\Async\ParallelError` when calling `await()` or `$pool->wait()`.
 
+### Working with tasks
+
+Besides using closures, you can also work with a Task. 
+A Task is useful in situations where you need more setup work in the child process.
+Because a child process is always bootstrapped from nothing, 
+chances are you'll want to initialise eg. the dependency container before executing the task.
```

Comment:
Why start on a newline here? Could it be that you don't have line wrapping configured in your editor?

Reply:
It's a habit I use for markdown files, because line wrapping in markdown files also enables it in PHP files, which I don't like. But I've added the changes.